src/app.py

```diff
@@ -1,13 +1,19 @@
 import pygame
+from Cards import Cards
 
 def main():
      
     pygame.init()
     
-    pygame.display.set_logo()
-    pygame.display.set_caption("Kopi-Cat Uno")
+    programIcon = pygame.image.load('./assets/kopicat.PNG')
+    
+    pygame.display.set_icon(programIcon)
+    pygame.display.set_caption("Kopi Cat")
 
     screen = pygame.display.set_mode((720,480))
+    
+    prototypeCard = Cards((255,0,0), 'normal')
+    prototypeCard.build(screen)
```

Comment:
we should make a build card function for the sake of code readability in the future.